mozilla/donate-wagtail
# Use the mailchimp-marketing API for thunderbird

Closes #1471 

## Steps for testing:
1. Checkout this branch
2. `inv new-env` (needed for the Mailchimp API package) 
3. Set your environment to be ThunderbirdDevelopment 
4. Add the `THUNDERBIRD_MC_API_KEY`, `THUNDERBIRD_MC_SERVER` and `THUNDERBIRD_MC_LIST_ID` to your local development settings 
5. Make a donation. It should no longer timeout (it should be nearly instant now) 
6. Make the same donation with the same email address, there should be no error (the mailchimp-marketing client will through an ApiClientError if the email address is already subscribed

If you want to test that it's actually works on local, you can add print statements between lines 788-814 in `donate/payments/views.py`
